.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{

```

Comment:
This should be gitignored, see comment on the \.gitignore

Reply:
Should it be? This seems helpful.

Reply:
it does but they also tried to get rid of it so

Reply:
no, i wanted to provide a default config for people, and one that they can change to their own without git getting in the way

Reply:
@jupyterkat make the gitignore `\.vscode` and `!\.vscode/launch.json` then

Reply:
you don't get the point, do ya, i don't want git to track launch.json

Reply:
but i do want it to CREATE launch.json